.github/workflows/checkbox-snap-daily-builds.yml

```diff
@@ -72,11 +80,12 @@ jobs:
             /home/runner/.local/state/snapcraft/log/
             checkbox-snap/series_${{ matrix.type }}${{ matrix.releases }}/checkbox*.txt
       - uses: actions/upload-artifact@v3
+        name: Upload snaps as artifact
         with:
           name: series_${{ matrix.type }}${{ matrix.releases }}
           path: checkbox-snap/series_${{ matrix.type }}${{ matrix.releases }}/*.snap
       - uses: Wandalen/wretry.action@a163f62ae554a8f3cbe27b23db15b60c0ae2e93c # v1.3.0
-        name: Upload checkbox snaps to the store
+        name: Upload Snap to the store
```

Comment:
the snap to make it clear we're talking about one, specified snap.

Reply:
And I think the command needs patching - no need for the loop now

Reply:
After re-merging them it does! The one in `checkbox-core-snap`  wouldn't need it, how should I substitute it? I would avoid calculating the name if possible

Reply:
Let's for now not mess with whimsical shell code.